src/browser-frontend/model/domain-page-meta-model.js

```diff
@@ -28,6 +28,7 @@ const DomainPageMetaModel = Immutable.Record({
   title: UNKNOWN_TITLE,
   favicon: UNKNOWN_FAVICON,
   bookmarked: false,
+  tabOwner: null,
```

Comment:
Uber nit: It might make some sense to not introduce the concept of "tabs" in domain state, as this is purely visual element. Something like `pageOwner` instead of `tabOwner` would be more consistent, but I don't want to overflow you with nits :)